src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/PanelMembersStatusReportITest.java

```diff
@@ -0,0 +1,265 @@
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardTableData;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/PanelMembersStatusReportITest_Typical.sql"
+})
+@SuppressWarnings({"PMD.LawOfDemeter", "PMD.JUnitTestsShouldIncludeAssert"})
+class PanelMembersStatusReportITest extends AbstractStandardReportControllerITest {
+
+    @Autowired
+    public PanelMembersStatusReportITest(TestRestTemplate template) {
+        super(template, PanelMembersStatusReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .trialNumber("111111")
+            .locCode(TestConstants.VALID_COURT_LOCATION)
+            .build());
+    }
+
+    @Test
+    void positiveTypicalCourt() {
+        testBuilder()
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated)
+            .assertEquals(getTypicalResponse());
+    }
+
+    @Test
+    void negativeUnauthorisedBureau() {
+        testBuilder()
+            .jwt(getBureauJwt())
+            .triggerInvalid()
+            .assertMojForbiddenResponse("User not allowed to access this report");
+    }
+
+    @Test
+    void negativeMissingTrialNumber() {
+        StandardReportRequest request = getValidPayload();
+        request.setTrialNumber(null);
+
+        testBuilder()
+            .payload(addReportType(request))
+            .jwt(getValidJwt())
+            .triggerInvalid()
+            .assertInvalidPathParam("trialNumber: must not be blank");
+    }
+
+    @Test
+    @SuppressWarnings({"CheckStyle.VariableDeclarationUsageDistance"})
+    void assertTotals() {
+        StandardReportResponse report = testBuilder()
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated).body();
+
+        long expectedPanelled = report.getTableData().getData().size();
+        long expectedEmpanelled = report.getTableData().getData().stream()
+            .filter(juror -> "Juror".equals(juror.get("panel_status"))).count();
+        long expectedReturned = report.getTableData().getData().stream()
+            .filter(juror -> "Returned".equals(juror.get("panel_status"))).count();
+        long expectedNotUsed = report.getTableData().getData().stream()
+            .filter(juror -> "Not Used".equals(juror.get("panel_status"))).count() + expectedReturned;
```

Comment:
Why do we add expectedReturned here?

Reply:
because the not used is the sum of both not used and returned jurors